etcdctl/ctlv3/command/printer_fields.go

```diff
@@ -30,12 +30,12 @@ func (p *fieldsPrinter) kv(pfx string, kv *spb.KeyValue) {
 	fmt.Printf("\"%sModRevision\" : %d\n", pfx, kv.ModRevision)
 	fmt.Printf("\"%sVersion\" : %d\n", pfx, kv.Version)
 	fmt.Printf("\"%sValue\" : %q\n", pfx, string(kv.Value))
-	fmt.Printf("\"%sLease\" : %d\n", pfx, kv.Lease)
+	fmt.Printf("\"%sLease\" : %016x\n", pfx, kv.Lease)
 }
 
 func (p *fieldsPrinter) hdr(h *pb.ResponseHeader) {
-	fmt.Println(`"ClusterID" :`, h.ClusterId)
```

Comment:
Can we cast it to `types.Id` to benefit from the common stringer implementation: 

(https://github.com/etcd-io/etcd/blob/53bfdc11d8f6d6cdf0ccb986f052e9739086110b/client/pkg/types/id.go#L22-L24) ?

Reply:
Thanks @ptabor . It makes sense. But only `memberID` and `clusterID` are uint64, and `LeaseID` is int64. So I only cast `memberID` and `clusterID` `to types.ID`.